src/client/entity-editor/relationship-editor/relationship-editor.tsx

```diff
@@ -355,6 +360,12 @@ class RelationshipModal
 		const relationships = generateRelationshipSelection(
 			relationshipTypes, baseEntity, otherEntity
 		);
+		if (baseEntity.type === 'Series') {
+			// When the base entity is Series, we need to remove relationshipType 70 - 74.
+			// As these relationships will be added via the series editor,
+			// we don't want to show them in the relationship modal.
+			_.remove(relationships, (relationship) => relationship.relationshipType.id > 69 && relationship.relationshipType.id < 75);
+		}
```

Comment:
Since this is essentially the same code as above, to avoid duplication I wonder if it can be extracted to a filter function similar to the existing `isValidRelationship` function in the same file.
That filter function can then be used both in `getValidOtherEntityTypes` and like this in `generateRelationshipSection`:

```
let validRels = candidateRelationships.filter(isValidRelationship);
if (baseEntity.type === 'Series') {
validRels = validRels.filter(isSeriesRelationship); /* <- The new reusable filtering method*/
}
```

Reply:
@MonkeyDo the reason to write this code explicitly like this was because now the `generateRelationshipSection` is also being used here https://github.com/bookbrainz/bookbrainz-site/pull/680/files#diff-ce680346708de8b052297af757840fc1df4f57226f9223ddc9c02d8c7af71d47R92-R94

If I add the following code to `generateRelationshipSection` function then it will remove relationshipType 70 - 74 for the new series editor component as well. 
```
let validRels = candidateRelationships.filter(isValidRelationship);
if (baseEntity.type === 'Series') {
validRels = validRels.filter(isSeriesRelationship); /* <- The new reusable filtering method*/
}
```

Reply:
I see, that makes sense, thanks :)

I also see now that the structure of the array we are filtering  is different (type.id vs relationship.relationshipType.id), so not an ideal candidate for refactoring.
In any case, the comments you added do help understand what is happening.